lib/engines/hcktest/hcktest.rb

```diff
@@ -151,21 +151,28 @@ def configure_clients
       end
     end
 
+    def reconfigure_clients
```

Comment:
This name does not represent what it actually does. "Reconfiguring" sounds like it is a special operation that corresponds to `configure_clients` but should be used for a second time. But it does more than configure (it also synchronizes). And such a sequence of configuring and synchronizing is also valid for the first run.